docs/api.md

```diff
@@ -81,156 +81,7 @@ These are the API group versions which are currently available when using Antrea
 
 | CRD | CRD version | Introduced in | Deprecated in | Removed in |
 |---|---|---|---|---|
+| `ClusterGroup` | v1alpha2 | v1.0.0 | v1.1.0 | v1.12.0 [^1] |
 
-## API renaming from `*.antrea.tanzu.vmware.com` to `*.antrea.io`
```

Comment:
It feels like you have these changes in another PR already?

Reply:
Yeah, I will rebase the code.

Reply:
Rebased.